requirements.txt

```diff
@@ -4,8 +4,8 @@ croniter >= 0.3.24, <1.0
 dask[bag] >=0.19.3, <3.0
 distributed >= 1.26.1, < 3.0
 docker >=3.4.1, <5.0
-marshmallow >= 3.0.0b19, < 4.0.0
-marshmallow-oneofschema >= 2.0.0b2, < 3.0
+marshmallow >= 3.0.0, < 4.0.0
+marshmallow-oneofschema >= 2.0.1, < 3.0
```

Comment:
Question @jlowin , why bump our lower bound if it still works?  This will potentially cause lots of environment breaks for people.

Reply:
That's a fair point @cicdw. I was a little uncomfortable leaving the prerelease bounds in place, but to your point -- if they work, they work. I'll revert.